src/guides/v2.3/coding-standards/docblock-standard-general.md

```diff
@@ -575,14 +578,17 @@ interface MutableInterface
  */
 class LimitedMutableClass implements MutableInterface
 {
+    /**
+    * Returns integer value
+    *
+    * @return int
```

Comment:
Same here:
> In general, typed method signature must be preferred over PHPDoc annotations whenever possible.

Reply:
@lenaorobei Thanks for the feedback. I have removed @return tag in PHPDoc.